app/models/ingredient.rb

```diff
@@ -1,3 +1,9 @@
 class Ingredient < ApplicationRecord
    validates_presence_of :name, :cost
+   has_many :recipe_ingredients
+   has_many :recipes, through: :recipe_ingredients
+
+   def self.sort_alphabetically
+      Ingredient.order(:name)
```

Comment:
This is an extension so it's not going to affect your score, but remember to test at the model level for this instance method as well as the feature level!